inst/automation/cronjob

```diff
@@ -1,5 +1,5 @@
 0 */5 * * * wget -O exclude_list.txt https://raw.githubusercontent.com/ropenscilabs/makeregistry/master/inst/automation/exclude_list.txt
```

Comment:
we're not using this anymore - its just  built on push and on the cron schedule set in the workflow files